templates/quat.rs.tera

```diff
@@ -124,7 +125,7 @@ pub struct {{ self_t }}(pub(crate) {{ simd_t }});
 
 impl {{ self_t }} {
     /// All zeros.
-    const ZERO: Self = Self::from_array([0.0; 4]);
+    pub const ZERO: Self = Self::from_array([0.0; 4]);
```

Comment:
This seem unrelated to the goal of this PR? A zero quaternion is an invalid quaternion, which I think is why I didn't make this `pub`. I am using it internally so perhaps it's useful to others, but unless it's necessary for this PR I think this should be reverted and discussed separately.